packages/app/src/cli/constants.ts

```diff
@@ -80,7 +80,12 @@ export const publicUIExtensions = {
 } as const
 
 export const uiExtensions = {
-  types: [...publicUIExtensions.types, 'pos_ui_extension', 'customer_accounts_ui_extension'],
+  types: [
+    ...publicUIExtensions.types,
+    'pos_ui_extension',
+    'customer_accounts_ui_extension',
+    'checkout_ui_extension_beta',
```

Comment:
Why are we calling this `checkout_ui_extension_beta`? Should this not be `ui_extension`?

Reply:
Agree with this 👍